### PR TITLE
New Landing Page 

### PR DESCRIPTION
**Pull request by Ishwarendra**
- soch samajhke kar diya
- Not responsive
- no snowfall, react can't find the module it was finding before idk why..

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-icons": "^4.11.0",
+    "react-snowfall": "^2.1.0",
```
Comment (harshjohar) [inline package.json:30]: remove this package, we dont need it probably

File: src/components/Landing.js [hunk 2]
```diff
@@ -2,35 +2,68 @@ import React from "react";
...
+  {
+    imgPath: "/assets/acm.png",
+    title: "Meet our team",
+    link: "https://www.pecacm.com/team",
+  },
+  {
+    imgPath: "/assets/acm.png",
+    title: "Meet our team",
+    link: "https://www.pecacm.com/team",
+  },
```
Comment (harshjohar) [inline src/components/Landing.js:68]: These items can be
- Meet our team (some icon of team)
- Continue to portal (pecacm.com) -- icon of pecacm
- Checkout leaderboard (pecacm.com/dashboard/leaderboard)

File: yarn.lock [hunk 3]
```diff
@@ -2503,6 +2508,13 @@ react-is@^18.2.0:
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-18.2.0.tgz#199431eeaaa2e09f86427efbb4f1473edb47609b"
   integrity sha512-xWGDIW6x921xtzPkhiULtthJHoJvBbF3q26fzloPCK0hsvxtPVelvftw3zjbHWSkR2km9Z+4uxbDDK/6Zw9B8w==
 
+react-snowfall@^2.1.0:
+  version "2.1.0"
+  resolved "https://registry.yarnpkg.com/react-snowfall/-/react-snowfall-2.1.0.tgz#b8c22c182c4e5cd084025c73169a3f399c3dd147"
+  integrity sha512-5nYWmBTJi/rrRRTmE62QLhxSp3sfX9niPP1Ysrd4dkVZLr0fxHTkcqCLQx7BAkclCEq5MWZ7kgIHrqk5Ugq1PA==
+  dependencies:
+    react-fast-compare "^3.2.2"
+
```
Comment (harshjohar) [inline yarn.lock:2517]: use `yarn remove react-snowfall` for clean removal

**Review by harshjohar — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Ishwarendra) [reply package.json]: Done ✅

**Review by harshjohar — APPROVED** [review]
LGTM

**Pull request merged**